.editorconfig

```diff
@@ -0,0 +1,21 @@
+# http://editorconfig.org
+
+root = true
```

Comment:
`root` means "Look no further! I am the last of my line!", which isn't really true here. For example, I have a `.editorconfig` in my homedir that provides some defaults, for projects which don't say anything. I think we shouldn't state this is the root.